.vsts.pipelines/jobs/prepare-artifacts.yml

```diff
@@ -0,0 +1,84 @@
+parameters:
+  gatherJobs: []
+  gatherPortableJob: ''
+
+  # downloadBuildConfig by default uses the current build, but can be changed to point at an
+  # existing/previous build by passing properties like these, instead:
+  #
+  #   buildType: specific
+  #   buildVersionToDownload: specific
+  #   project: 'public'
+  #   definition: 'source-build-CI'
+  #   buildId: 741400
+  #
+  # By defining this in one place, it's easier to set this up if necessary. This may be useful
+  # because the build takes a long time relative to prepare-artifacts, and we may want to tweak
+  # prepare-artifacts in case it fails due to a bug in the future. ...It's also nice to have the
+  # properties listed in this comment even if we don't use them here, because it's hard to figure
+  # them out from the docs alone--it'll be a nice reference.
+  downloadBuildConfig:
+    buildType: current
+
+jobs:
+- job: PrepareArtifacts
+  pool:
+    vmImage: 'ubuntu-18.04'
+  dependsOn:
+  - ${{ each gather in parameters.gatherJobs }}:
+    - ${{ gather.job }}
+  timeoutInMinutes: 300
```

Comment:
This seems a bit excessive for just gathering artifacts but if it's what we're already using I don't have a problem with it.